example/src/App.tsx

```diff
@@ -74,7 +75,7 @@ const exampleCode = {
 
 const App = () => {
   const [activeTab, setActiveTab] = useState(tabs.SINGLE_TRANSACTION);
-  const { batches, estimate, send } = useEtherspotUi();
+  const { batches, estimate, send, smartWalletAddresses } = useEtherspotUi();
```

Comment:
would rename `smartWalletAddresses` as `getSmartWalletAddresses` or similar because it's: 1) function 2) returns promise

Reply:
Agreed, i actually thought of this too but my eyes were closing 😂

Reply:
Done 👍